parachain/primitives/core/src/outbound.rs

```diff
@@ -278,6 +278,14 @@ pub trait SendMessage {
 	fn deliver(ticket: Self::Ticket) -> Result<H256, SendError>;
 }
 
+/// A trait for getting the local costs associated with sending a message.
+pub trait OutboundQueueLocalFee {
+	type Balance: BaseArithmetic + Unsigned + Copy;
+
+	/// Calculate fee in native currency for processing a message locally
+	fn calculate_local_fee() -> Self::Balance;
```

Comment:
```suggestion
	fn local_fee() -> Self::Balance;
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/999/commits/be36fbe3a8f0b9f9478a900a479d5e59e8fdd12d